lib/fetch/file.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
+
+const { Blob } = require('buffer')
+const { kState } = require('./symbols')
+
+class File extends Blob {
```

Comment:
FWIW, I'm planning to have `File` in core along side `Blob`. Was looking to add it within the next few weeks.

Reply:
Excellent!

Reply:
Then we can probably remove this before going out of experimental.

Reply:
@jasnell any chance you would consider doing `FormData` as well? Feel free to copy paste from this PR.

Reply:
This is what happened to fetch-blob, We only had a Blob class first, then everyone else where creating there own File class and extended it. So that's why we added a File class to fetch-blob as well. they kind of go hand in hand